versions.tf

```diff
@@ -4,7 +4,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = "~> 3.0"
+      version = "~> 4.0"
```

Comment:
if the module is technically compatible with both versions 3 and 4, what about `>= 3, < 5`?

Reply:
@jansiwy This would then probably be a minor version bump, I assume?